code/modules/reagents/machinery/chem_master.dm

```diff
@@ -78,6 +79,8 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 	max_bottle_size = initial(max_bottle_size) + lasercount*5
 	max_pill_count = initial(max_pill_count) + manipcount*5
 	handle_new_reservoir(scancount*25+100)
+	max_pill_size = initial(max_pill_size)+manipcount*25 // i suck at math
```

Comment:
earlier in the code at line 73 it subtracts 1 level from the manip so T1 would be worth 0, T2, is 1, and T3 is 2 with t2 being 1  1*25+50=75 and t3 being 2  2*25+50=100